app/assets/templates/poll_tpl.jst.hbs

```diff
@@ -13,13 +13,26 @@
       {{#poll.poll_answers}}
         <div class="result-row">
           <input type="radio" name="vote" value="{{id}}"/>
-          <label>{{answer}}</label>
+            {{#if isCurrentUserVote}}
+              <label>
+                {{answer}}
+                  {{#linkToAuthor currentUser}}
+                    {{{personImage this 'small' 'micro'}}}
+                  {{/linkToAuthor}}
+              </label>
+            {{else}}
+              <label>{{answer}}</label>
+            {{/if}}
```

Comment:
The indentation here is a bit weird. `{{#if isCurrentUserVote}}` should be indented like `<input type="radio" name="vote" value="{{id}}"/>` and both `{{answer}}` and `{{#linkToAuthor currentUser}}` like `<label>`.